src/Backoffice/Airline/App/Controllers/DeleteAirlineController.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Airline\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Lightit\Backoffice\Airline\Domain\Actions\DeleteAirlineAction;
+use Lightit\Backoffice\Airline\Domain\Models\Airline;
+
+class DeleteAirlineController
+{
+    public function __invoke(DeleteAirlineAction $deleteAirlineAction, Airline $airline): JsonResponse
```

Comment:
```suggestion
    public function __invoke(DeleteAirlineAction $action, Airline $airline): JsonResponse
```
Since you have a single action, you don't need to be overly specific.

Reply:
This is more of a suggestion, not a standard, it’s not a problem to have a long name.